### ISSUE TITLE
How can I remove the (blue) overhang detection

### ISSUE THREAD

**Issue by AndyCap777** [supermerill/SuperSlicer#1957]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe how it would work**
Be as descriptive as your skill level allows you.

**Describe alternatives you've considered**

**Additional context**
Add any other context or paint drawing about the feature request here.

**Comment by AndyCap777** [comment]
The blue overhang detection is messing up my model. How can I switch it off and use normal support.

**Comment by neophyl** [comment]
Please attach the requested 3mf project file zipped up. With that we can see exactly what your issue is, possibly modify the settings and post the altered one back.

**Comment by AndyCap777** [comment]
I am not sure why you want me to do all that. All I am asking is how do you switch off the "Overhang Perimeter" It's shown as blue in the "Feature Type" bog in the top left corner.

**Comment by neophyl** [comment]
It’s not quite that simple. First you have opened an issue, so are reporting a bug, those ask for an example project so we can see the issue.  Regardless of that you haven’t even included a picture indicating what exactly you want to switch off. 

Any answer given will be a guess as you haven’t provided needed information. I try and answer questions if I can but I dont like guessing unless it’s unavoidable. 

Dark blue is classed as overhang. Unless you mean the light blue ? That’s bridge infill. Which blue do you mean ?  Anything classed is overhang/bridging gets printed with a layer height the same as the nozzle width. This is because technically it’s not squished into the layer below so it’s treated as being a round extrudate rather than being the layer height. It’s one of the annoying things about slicer. 
If my ASSUMPTION is correct then If I had to guess then I would suggest you change the settings around overhang angle and flow %.  But it differs depending on the model.  With a project we could just see and make changes, see if the suggestion works for this instance and post a modified project back. You could then open it and see what was done to modify it. 

Basically including a project from the start streamlines the whole process which is why the issue template asks for one.

**Comment by macchinette** [comment]
Set the overhangs_width_speed and overhangs_width parameters to zero.

![image](https://user-images.githubusercontent.com/79080486/143291021-7b755ff1-c22b-469e-b577-0ddab37cb9ea.png)

**Comment by AndyCap777** [comment]
Thanks Macchinette It worked
